build.sbt

```diff
@@ -453,8 +453,8 @@ lazy val kernelApi = (project in file("kernel/kernel-api"))
     libraryDependencies ++= Seq(
       "org.roaringbitmap" % "RoaringBitmap" % "0.9.25",
       "org.slf4j" % "slf4j-api" % "1.7.36",
+      "com.fasterxml.jackson.core" % "jackson-databind" % "2.13.5",
```

Comment:
We intentionally made the `kernel-api` module light-weight by no depending on any other libraries. We make use of the JsonHandler for any JSON parsing-related stuff. Can we use it here? in

Reply:
Will add an API in JsonHandler to deserialize string in to map for TableConfig.

Reply:
We need to discuss a little bit. Because the `TableConfig` need to access engine in this way.

Reply:
The one-page decision [doc](https://docs.google.com/document/d/1vdK2nXubEUlO8FriI7gFsEtLvBFMAMZlXzXSopqMJiI/edit?usp=sharing) for the discussion.